### PR TITLE
Added apply stimulus function.

### PR DESCRIPTION
**Pull request by jeffgege**
Hi,
I added a function in schema.py to apply a stimulus to an iTop object.

### CODE REVIEW

File: itoptop/schema.py [hunk 1]
```diff
@@ -292,4 +292,38 @@ def lookup(self, obj):
 
         return obj
 
+    def apply_stimulus(self, query, stimulus_data, stimulus="ev_assign"):
```
Comment (jonatasrenan) [inline itoptop/schema.py:295]: typo ev_assign

**Review by jonatasrenan — APPROVED** [review]

**Comment by jonatasrenan** [comment]
Thanks @jeffgege

**Comment by jonatasrenan** [comment]
https://pypi.org/project/itoptop/1.0.10/

**Comment by jeffgege** [comment]
Thank you @jonatasrenan!

**Pull request merged**